packages/suite/src/views/firmware/FirmwareCustom.tsx

```diff
@@ -76,6 +76,7 @@ const FirmwareCustom = () => {
         <Modal
             cancelable={isCancelable}
             onCancel={onClose}
+            noSidePadding={status !== 'initial'} // TODO: redesign custom fw modal to match fw update modal
```

Comment:
is there any issue for this TODO? issues are easier to manage (prioritise) than TODOs, so would be nice to add link to issue...

Reply:
#4261

Reply:
nice 👍 consider placing the link directly into the comment in the code